### PR TITLE
fix bug with two faces

### PR DESCRIPTION
**Pull request by AndreiEvstratenko**
# Pull Request
fix bug with two faces
## Description

- remove existing nodes

### Motivation and Context

- closes #1829

### Types of changes

- [x] Bug fix
- [ ] New feature
- [ ] Maintenance

### CODE REVIEW

File: examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/Classes/Controllers/ViewController.swift [hunk 1]
```diff
@@ -118,9 +118,10 @@ extension ViewController: SearchCollectionViewDelegate {
    
     func mediaSelected(media: SvrfMedia) {
```
Comment (brentchow) [inline examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/Classes/Controllers/ViewController.swift]: unnecessary whitespace

File: examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/Classes/Helpers/VirtualFaceContent/RemoteFaceFilter.swift [hunk 2]
```diff
@@ -38,6 +38,14 @@ class RemoteFaceFilter: SCNNode, VirtualFaceContent {
         }
     }
     
+    func resetFaceFilters() {
+        if let head = self.faceFilter {
+            for child in head.childNodes {
+                child.removeFromParentNode()
```
Comment (brentchow) [inline examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/Classes/Helpers/VirtualFaceContent/RemoteFaceFilter.swift]: does this break if there aren't any childNodes?

File: examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/Classes/Controllers/ViewController.swift [hunk 3]
```diff
@@ -118,9 +118,10 @@ extension ViewController: SearchCollectionViewDelegate {
...
         
-            activityIndicator.startAnimating()
-            remoteFaceFilter.loadFaceFilter(media: media)
-            resetButton.isHidden = false
+        remoteFaceFilter.resetFaceFilters()
```
Comment (brentchow) [inline examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/Classes/Controllers/ViewController.swift]: Please add a comment explaining that: 

```swift
// Remove any existing face filter from the SCNScene
```

File: examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/Classes/Helpers/VirtualFaceContent/RemoteFaceFilter.swift (hunk 2 above)
Comment (AndreiEvstratenko) [reply examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/Classes/Helpers/VirtualFaceContent/RemoteFaceFilter.swift]: If we have no childNodes in our head we just don't enter in cycle, so, everything is ok

File: examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/Classes/Controllers/ViewController.swift (hunk 1 above)
Comment (AndreiEvstratenko) [reply examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/Classes/Controllers/ViewController.swift]: done

File: examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/Classes/Controllers/ViewController.swift (hunk 3 above)
Comment (AndreiEvstratenko) [reply examples/ARKitFaceFilterDemo/ARKitFaceFilterDemo/Classes/Controllers/ViewController.swift]: done
